fluss-common/src/main/java/org/apache/fluss/metadata/DataLakeFormat.java

```diff
@@ -21,7 +21,8 @@
 public enum DataLakeFormat {
     PAIMON("paimon"),
     LANCE("lance"),
-    ICEBERG("iceberg");
+    ICEBERG("iceberg"),
+    VALUES("values");
```

Comment:
I'm still concerned on introduing a format just for testing. 
Actaully, we don't need to introduce a new one format. 
We already have a `org.apache.fluss.server.lakehouse.TestingPaimonStoragePlugin` for test purpose. I think we can resue `TestingPaimonStoragePlugin`?

fluss-flink module introduce fluss-server test module which contains `TestingPaimonStoragePlugin`